moped-api/README.md

```diff
@@ -39,7 +39,7 @@ $ source venv/bin/activate
 3.Now you are ready to install the requirements
 
 ```
-$ pip install -r requirements/dev.txt
+$ pip install -r requirements/development.txt
```

Comment:
Not a problem at all, just curious about the rename :)

Reply:
Forgot to mention that in the notes, but I had noticed in the past that there was a mismatch between the README steps and the actual file name. Just snuck that update in here.